src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx

```diff
@@ -16,36 +16,86 @@ redirects:
   - /docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-best-practices
 ---
 
-This document describes some best practices and examples to help you get the most out of your alerts configuration.
+Improve your alert coverage by implementing the following recommendations. 
+
+Read on to learn the best practices for:
+- [Policies](/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices/#policy-practices)
+- [Notification channels](/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices/#channel-practices)
+- [Incident preferences](/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices/#incident-practices)
+- [Thresholds and violations](/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices/#threshold-practices)
+- [Muting roles](/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices/#mute-practices)
 
 <Callout variant="important">
-  Before reading this, we recommend you read [Alerts concepts and workflow](/docs/alerts/new-relic-alerts/getting-started/alert-policy-workflow).
+  Before reading this, we recommend you also read [Alerts concepts and workflow](/docs/alerts/new-relic-alerts/getting-started/alert-policy-workflow).
 </Callout>
 
-## Define policies for entities or people [#entities-people]
+## Recommended alerts [#recommend-alerts]
+
+Use [recommended practices](https://discuss.newrelic.com/t/announcing-alert-recommendations-for-apm-applications/154542) if you are new to Alerts or if you want suggestions that optimize your alert coverage.
+
+## Policies [#policy-practices]
+
+A policy is a container for alike conditions.
+
+<Callout variant="tip">
+Learn how to [create, edit, or find policies](/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/create-edit-or-find-alert-policy/) if you’re new to Alerts.
+</Callout>
 
-When designing your policies, consider:
+Organize your policy by surrounding its conditions to a single entity and to the specific team that needs to be notified when a problem arises within that entity.
 
-* The parts of your architecture that need personnel to be responsible for them
-* The individuals who are responsible for one or more parts of your infrastructure
+If a team is monitoring several groups of the same entity type, combine those entity clusters (like servers) together into one policy. This way, your team can be notified from one policy rather than navigating several policies at once.
 
-An organization may have multiple entities monitored by APM, browser, infrastructure, and synthetic monitoring. Examples of considerations for different teams:
+Consider your team’s role when assigning them to policies:
```

Comment:
I think you may be losing some useful detail in this rewrite....

Also, it's not the "team's" role that we want to consider, right? More the individual on a team? Or maybe it's a combination of team and individual roles?